src/Framework/Microsoft.Build.Framework.csproj

```diff
@@ -8,6 +8,7 @@
     <RootNamespace>Microsoft.Build.Framework</RootNamespace>
     <AssemblyName>Microsoft.Build.Framework</AssemblyName>
     <EnableDocumentationFile>true</EnableDocumentationFile>
+    <UpdateNuGetPackageDependencies>true</UpdateNuGetPackageDependencies>
```

Comment:
I tried to do this but `<Import />` doesn't let me use an item group in a condition.  I'll leave this for now but will probably remove it in the near future.